jena-geosparql/src/test/java/org/apache/jena/geosparql/spatial/SpatialIndexTestData.java

```diff
@@ -60,21 +71,28 @@ public class SpatialIndexTestData {
     public static final SRSInfo WGS_84_SRS_INFO = new SRSInfo(SRS_URI.WGS84_CRS);
     public static final SRSInfo OSGB_SRS_INFO = new SRSInfo(SRS_URI.OSGB36_CRS);
 
-    private static SpatialIndex TEST_SPATIAL_INDEX = null;
+    private static SpatialIndexPerGraph TEST_SPATIAL_INDEX = null;
     private static Dataset TEST_DATASET = null;
 
-    public static final SpatialIndex createTestIndex() {
+    public static final List<SpatialIndexItem> getTestItems() {
+        List<SpatialIndexItem> items = List.of(
+                SpatialIndexItem.of(LONDON_GEOMETRY_WRAPPER.getEnvelope(), LONDON_FEATURE),
```

Comment:
5 deprecated functions here. Add `.asNode()` to second argument.

Reply:
Added `.asNode()` as suggested.